concourse/create-master-build-configs/task.yml

```diff
@@ -0,0 +1,15 @@
+---
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: node
+
+inputs:
+- name: rex-web 
+outputs:
+- name: build-configs
+
+run:
+  path: rex-web/concourse/create-master-build-configs/script.js 
```

Comment:
Is it possible to just use `./script.js` (in case any parent directories are renamed)

Hmm, concourse probably does not though. But maybe worth checking?

Reply:
no, the working directory will be the build folder, this is just the config file it loads